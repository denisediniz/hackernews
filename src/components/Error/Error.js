import React from 'react';
import styles from './index.module.scss';

const Error = () => (
  <div className={`${styles.columns} ${styles['is-centered']}`}>
    <div className={`${styles.column} ${styles['is-narrow']}`}>
      <div
        className={`${styles.notification} ${styles['is-danger']} ${styles['is-light']}`}
      >
        Something went wrong.
      </div>
    </div>
  </div>
);

export default Error;
