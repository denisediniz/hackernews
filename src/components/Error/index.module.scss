@import 'bulma/sass/utilities/_all.sass';
@import 'bulma/sass/elements/notification.sass';
@import 'bulma/sass/grid/columns.sass';
