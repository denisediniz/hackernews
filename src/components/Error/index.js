import Error from './Error';
import withError from './withError';

export default Error;
export { withError };
