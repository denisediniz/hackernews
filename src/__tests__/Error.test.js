import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Error from '../components/Error';

describe('Error', () => {
  it('renders without crashing', () => {
    shallow(<Error />);
  });

  it('has a valid snapshot', () => {
    const component = renderer.create(<Error />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
