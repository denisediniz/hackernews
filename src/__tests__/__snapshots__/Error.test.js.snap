// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error has a valid snapshot 1`] = `
<div
  className="columns is-centered"
>
  <div
    className="column is-narrow"
  >
    <div
      className="notification is-danger is-light"
    >
      Something went wrong.
    </div>
  </div>
</div>
`;
